apps/experiments/views/experiment.py

```diff
@@ -1006,7 +1006,7 @@ def experiment_review(request, team_slug: str, experiment_id: str, session_id: s
             "survey_link": survey_link,
             "survey_text": survey_text,
             "form": form,
-            "available_tags": [t.name for t in Tag.objects.filter(team__slug=team_slug).all()],
+            "available_tags": [t.name for t in Tag.objects.filter(team__slug=team_slug, is_system_tag=False).all()],
```

Comment:
Not blocking but you could make this a model manager method since it's a common use case